dev/lint-python

```diff
@@ -60,9 +60,9 @@ export "PYLINT_HOME=$PYTHONPATH"
 export "PATH=$PYTHONPATH:$PATH"
 
 # There is no need to write this output to a file
-#+ first, but we do so so that the check status can
-#+ be output before the report, like with the
-#+ scalastyle and RAT checks.
```

Comment:
Ah, this wasn't from SPARK-23174 tho.

Reply:
Hmmm .. seems this indicates line continuation and intentional .. ?

Reply:
Is that so?  We have 100 character limit on a single line according to the style guide.  Maybe all 4 lines could be rearranged?

Reply:
Hmm .. @nchammas, some of these seem added by you a long ago. Could you take a look for this one please? I think I am not aware of this. Wonder if we can just remove those.

Reply:
The `#+` convention is something I picked up from [the Linux Documentation Project](http://tldp.org/LDP/abs/html/here-docs.html#COMMENTBLOCK), if that's what you're referring to. You can safely do away with it and just have the `#`.

It was a "phase". I'm over it now... 😆

Reply:
Wow, thank you so much.